DangerSwap/DangerSwap/Infrastructure/StringLengthConst.cs

```diff
@@ -1,5 +1,6 @@
 ﻿namespace DangerSwap.Infrastructure
 {
+    //Mindaugas
```

Comment:
Lines 6-11 const lines should not be started lower case if it's not private field. I'd suggest to rename them with UpperCase names.